modules/core/src/main/java/org/apache/ignite/internal/client/thin/ClientUtils.java

```diff
@@ -86,154 +80,17 @@ static int cacheId(String name) {
         return name.hashCode();
     }
 
-    /**
-     * @param col Collection to serialize.
-     * @param out Output stream.
-     * @param elemWriter Collection element serializer
-     */
-    public static <E> void collection(
```

Comment:
Why do you move this method to BinaryUtils? It used only in client context.

Reply:
IMHO, it is more obvious, when it is "single place of truth" for `#collection` methods, but not clear, when one method will be in `BinaryUtils`, second others in `ClientUtils`